### DISCUSSION TITLE
Changing text fields

### DISCUSSION THREAD

**Discussion by mjh8431** [RafaelGB/obsidian-db-folder#289] [Q&A] [answered]
I just added close to 200 book notes from GoodReads using the Booksidian plugin. Because of their RSS limitations I had to split one of the categories, which shows the status "Read" into two categories "Read1" and "Read2". I want to use DB Folder to manage these notes. Is there any way to bulk change the text in this status field with DB Folder? Or is my only option in DB Folder to manually change the status for each file?

This is a great plugin. Thanks for your work.

**Comment by RafaelGB** [comment, marked as answer]
Currently is not possible to manage massive changes. You have to manually modify every row.

There is an issue on the roadmap that maybe could fix your use case: https://github.com/RafaelGB/obsidian-db-folder/issues/129

But I have no plans to implement a native way to do that. We can discuss to add a long-term feature for that
